starcheck/src/starcheck.pl

```diff
@@ -49,21 +49,43 @@
 use Inline Python => q{
 
 import os
+import six
 from chandra_aca.star_probs import set_acq_model_ms_filter
 import starcheck
 from starcheck.pcad_att_check import make_pcad_attitude_check_report, check_characteristics_date
 from starcheck.calc_ccd_temps import get_ccd_temps
 from starcheck.version import version
 
+def debyte_kwargs(kwargs):
+    if six.PY2:
```

Comment:
Did we decide we're ready to have the code by Py3 only?

Reply:
Let's plan on Py3 only.